framework/wazuh/cluster/local_client.py

```diff
@@ -99,12 +92,7 @@ def __init__(self, command: bytes, data: bytes, wait_for_complete: bool):
 
     async def send_api_request(self):
         result = (await self.protocol.send_request(self.command, self.data)).decode()
-        if result.startswith('Error'):
-            raise exception.WazuhException(3009, result)
-        elif result.startswith('WazuhException'):
-            _, code, message = result.split(' ', 2)
-            raise exception.WazuhException(int(code), message)
-        elif result == 'There are no connected worker nodes':
+        if result == 'There are no connected worker nodes':

```

Comment:
This could be modeled using a `WazuhClusterError` instead of hardcoding message.

Reply:
I didn't see that one as an exception tbh. That message comes from executing a _broadcasting_ request in a single node environment:
https://github.com/wazuh/wazuh/blob/f75c8e3c079433a36542043f4500dd4c58884b04/framework/wazuh/cluster/dapi/dapi.py#L333-L334
https://github.com/wazuh/wazuh/blob/f5326f7c3684c77d012a6b103f5e70914e575f95/framework/wazuh/cluster/local_server.py#L107-L113

But as you can see, the response command returned is an `ok`  and not an `err`. That's why I don't consider this an exception.

Reply:
I see, you mean it is a valid response from local_server. I think it makes sense. Thanks for clarifying.